samples/Playground/App.fs

```diff
@@ -86,5 +86,9 @@ module App =
         DesktopApplication(Window(content()))
 #endif
 
+#if DEBUG
+            .attachDevTools()
```

Comment:
Tried this and I can not see the DevTool window. I think we need to wrap the Diagnostics package.

Reply:
Hm - works for me on this project too, with the same config as mentioned above.

Reply:
Thanks. really nice to have this modifiers 👍